orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -1003,6 +1003,39 @@ def run_step(model, input):
 
         _test_helpers.assert_values_are_close(ort_prediction, pt_prediction)
 
+@pytest.mark.parametrize("operator", ['min', 'max'])
+@pytest.mark.parametrize("dim", [None, 0, -1])
+@pytest.mark.parametrize("keepdim", [True, False])
```

Comment:
Better also test keepdim is None (i.e., Max(x,y)) case, maybe add a new test case for it. As you also move such case from PyTorch exporter to your above code.